### DISCUSSION TITLE
Getting a blank page upon hitting the init url

### DISCUSSION THREAD

**Discussion by NithinMahesh1** [mixcore/mix.core#735] [Q&A]
Need some help, I had this almost running the other day on a Windows 2022 server with IIS 10. I have the mixcore-v2.0.0-alpha.1 files placed in a the directory: "C:/inetpub/wwwroot/MixCore" and on iis configured as an application on the "Default Web Site" (set to port 90) so the url to access was http://localhost:90/MixCore/init.

With this configuration, loading the same url, I was able to get to what I assumed was a pre created theme with the text "Page : Home". I also realized my database was pre populated from a bad install so I reset my database. I am now trying to reconfigure with this setup again, but with the same configurations and url I see a blank page and the url redirects to http://localhost:90/init. If I look at the console on my browser I see I get 404's for index.html and view.html

Someone please help!

**Comment by Smilefounder** [comment]
@nhathoang989 @truongtphat any ideas?
